### PR TITLE
[FEATURE] Add Pet

### PR DESCRIPTION
**Pull request by alexanderbarrosorellana**
- Modify the linter implementation
- Change initial screen to  be Groups
- Update Gradle
- Modify axios interceptor
- List pets and users in group cards
- Add Create Pet screen
- Add MockedNavigator to testing utils

 ### To accomplish a smoother flow while creating a group or/and a pet, also it is to maintain some conventions by using the linter 

>NOTE: sorry to my reviewer because there are too many changes that are just semicolon ( ; ) removals

### CODE REVIEW

File: src/components/group-card/MemberList.js [hunk 1]
```diff
@@ -1,38 +1,36 @@
-import React from 'react';
-import { Icon, List, ListItem } from '@ui-kitten/components';
+import React from 'react'
+import { Icon, List, ListItem } from '@ui-kitten/components'
 // import ButtonAccesory from './ButtonAccessory';
```
Comment (MatiasBarrera) [inline src/components/group-card/MemberList.js:4]: Quitar? o no?

File: src/lang/es.json [hunk 2]
```diff
@@ -15,11 +16,25 @@
   },
   "groupList": {
     "create": "Crear Grupo",
+    "empty": "Todavía no hay grupos, puedes crear uno persionando el botón de abajo",
```
Comment (MatiasBarrera) [inline src/lang/es.json]: "persionando"

File: src/lang/es.json [hunk 3]
```diff
@@ -15,11 +16,25 @@
...
+    "error": "Algo salió mal, intenta mas tarde o contáctate con nosotros",
```
Comment (MatiasBarrera) [inline src/lang/es.json]: en vez de "contáctate con nosotros" creo que suena mejor  "contáctanos"

File: src/screens/groups/GroupList.js [hunk 4]
```diff
@@ -68,22 +95,26 @@ const GroupList = ({ navigation }) => {
...
-            style={themedStyles.createGroupButton}>
-            {groupList.create}
-          </Button>
-        </SafeAreaView>
+        </>
       )}
-    </View>
-  );
-};
+      {/* Temporary button to sign out, this should be removed with bottom navigation implementation */}
```
Comment (MatiasBarrera) [inline src/screens/groups/GroupList.js]: En un "//TODO:" o "//FIXME:" (?)

**Review by MatiasBarrera — CHANGES_REQUESTED** [review]
🚀 Nice job... unas observaciones por ahi 👀

File: src/components/group-card/MemberList.js (hunk 1 above)
Comment (alexanderbarrosorellana) [reply src/components/group-card/MemberList.js:4]: no, es por el comentario de abajo, copy pastearé

**Review by MatiasBarrera — APPROVED** [review]
🐰

**Pull request merged**